internal/commands/root/root.go

```diff
@@ -201,6 +204,9 @@ func runRootCommandWithProviderAndClient(ctx context.Context, pInit provider.Ini
 	if err != nil {
 		return errors.Wrap(err, "error setting up pod controller")
 	}
+	if podCtrl != nil {
+		*podCtrl = *pc
```

Comment:
Here, and similar with `nodeCtrl`, it is assigning the value of `pc` to the value of `podCtrl`.

This is where the linting error comes from.

Reply:
Already fix this issue.